docs/source/index.rst

```diff
@@ -69,6 +69,7 @@ model implementations and application components.
    tutorials/online_asr_tutorial
    tutorials/device_asr
    tutorials/ctc_forced_alignment_api_tutorial
+   tutorials/forced_alignment_for_multilingual_data_tutorial
```

Comment:
Please move the original tutorial to `API Tutorials` section.

Reply:
sure